examples/embedding/embedding.cpp

```diff
@@ -24,11 +24,11 @@ int main(int argc, char ** argv) {
 
     fprintf(stderr, "%s: build = %d (%s)\n", __func__, BUILD_NUMBER, BUILD_COMMIT);
 
-    if (params.seed < 0) {
+    if (params.seed == -1) {
```

Comment:
Wouldn't this cause compiler warnings?
I think better replace `-1` with `0xFFFFFFFF` everywhere, except in the help and the readme